BraintreeModels/src/main/java/com/braintreepayments/api/Authorization.java

```diff
@@ -22,18 +20,21 @@
      *
      * @param authorizationString Given string to transform into an {@link Authorization}.
      * @return {@link Authorization}
-     * @throws InvalidArgumentException This method will throw this exception type if the string
```

Comment:
Same comment here - can we provide some info in this docstring for how an invalid string will surface as an error?

Reply:
Added more detail to the description of this method 👍